packages/peregrine/lib/talons/ForgotPassword/useForgotPassword.js

```diff
@@ -1,37 +1,53 @@
 import { useCallback, useState } from 'react';
+import { useMutation } from '@apollo/react-hooks';
 import { useUserContext } from '@magento/peregrine/lib/context/user';
 
 /**
  * Returns props necessary to render a ForgotPassword form.
- * @param {function} props.onClose callback function to invoke when closing the form
+ *
+ * @param {function} props.onCancel - callback function to call when user clicks the cancel button
+ * @param {function} props.mutations.resetPasswordMutation - mutation to call when the user clicks the submit button
+ *
+ * @returns {{
+ *  formErrors: [Error],
+ *  forgotPasswordEmail: string,
+ *  inProgress: boolean,
+ *  isResettingPassword: boolean,
+ *  handleCancel: function,
+ *  handleFormSubmit: function,
+ * }}
```

Comment:
Now might be a good opportunity to use the doc style @jcalcaben went over today.